.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+-   repo: https://github.com/ambv/black
```

Comment:
please take an example in pytest wrt other tools to add (like pyupgrade, eol fixing, whitespace fixing, )

Reply:
Nice @RonnyPfannschmidt! This is all new to me but will definitely come in handy! I had tried to look into this for work, but found nothing. Which is odd, but I was trying to find the eq to JS prettier.

@davehunt Yeah, I will explore the different options to see if there's anything we can benefit from.